lab4_controller.py

```diff
@@ -167,13 +167,13 @@ def automatch_phab(self, lab, match=1):
                 width = self.scan_width(scanNum)
                 if width < 200 or width > 4000:
                     # not working
-                    self.logger.warning(f"Delay line not working (width {width}), trying to kick")
+                    self.logger.warning(f"LAB{i:<2}: Delay line not working (width {width}), trying to kick")
                     self.l4reg(lab, 8, 2500)
                     time.sleep(0.1)
                     width = self.scan_width(scanNum)
-                    self.logger.warning(f"Width now {width}")
+                    self.logger.warning(f"LAB{i:<2}: Width now {width}")
                     if width < 200 or width > 4000:
-                        self.logger.error("still not working, bailing")
+                        self.logger.error(f"LAB{i:<2}: Still not working, bailing")
                         err = True
                         continue
                     self.l4reg(lab, 8, 2700)
```

Comment:
What is this line `self.l4reg(lab, 8, 2700)` doing there? It looks like it changes the register only if the `width` is in the desired range (but also only after it failed the first if condition) ...

Reply:
@ryankrebs016 and @cozzyd does it for you make sense to "kick" the labs all with a different value at the end?